pinot-common/pom.xml

```diff
@@ -62,6 +62,27 @@
         <artifactId>protobuf-maven-plugin</artifactId>
       </plugin>
 
+      <plugin>
+        <groupId>com.diffplug.spotless</groupId>
+        <artifactId>spotless-maven-plugin</artifactId>
+        <configuration>
```

Comment:
We usually keep the plugin management inside the root pom, and then we can simply use it here

Reply:
I think this change is fine. Basically he moved this from pluginManagement to plugin. Given there is no child project that inherits its pom, I think this is the right move.

Reply:
Yeah, I assumed that https://github.com/apache/pinot/pull/11670 introduced the `pluginManagement` in this POM for the JDK 21 build profile (i.e., to avoid running this plugin's goal with Java 21)? Also we're overriding the plugin configuration here (for the separate excludes I presume), so it's not like we can simply define it here without the configuration right?

Reply:
What I meant is that we should be able to put the `excludes` in the root pom where there is another configuration for this plug-in, then we keep a single configuration for this plug-in for easier management

Reply:
Ah okay, I've made that change. Not sure why it was like that earlier - perhaps to avoid adding child module specific excludes in the parent POM? Looks like it was added in https://github.com/apache/pinot/pull/8427/files.

Reply:
> What I meant is that we should be able to put the excludes in the root pom where there is another configuration for this plug-in, then we keep a single configuration for this plug-in for easier management

I think that is a bad practice. This is the project that knows which files should be ignored. In case other projects want to do the same we don't want to merge that information.

The pom project should specify the plugin version and the _common_ configuration. Configuration specific to each sub-project should be configured by each sub-project

Reply:
Makes sense, I've reverted the last commit to keep things as they were previously.

Reply:
@gortiz Does it work if we only keep the `excludes` config here? Keeping separate `excludes` per module definitely make sense, but replicating other fields can add management overhead

Reply:
Based on https://maven.apache.org/pom.html#plugins, that should work and it is supported behavior. I've pushed a commit making this change and also verified locally that it works as expected.